utm.yaml

```diff
@@ -60,32 +62,34 @@ components:
       type: oauth2
       flows:
         clientCredentials:
-          tokenUrl: https://token_vendor.com/oauth/token
+          tokenUrl: https://auth.example.com/oauth/token
           scopes:
             utm.strategic_coordination: |-
-              Client may perform planning, strategic conflict detection, and conformance monitoring activities.
+              Client may perform operations encompassed by the strategic coordination role including strategic conflict detection.
```

Comment:
Can we replace "operations" with another word, e.g., "functions"? We refer to operations in the operational intent context a lot in this spec - just want to delineate clearly to avoid potential confusion.

Reply:
Replaced with "actions"